forge-game/src/main/java/forge/game/phase/PhaseHandler.java

```diff
@@ -1050,12 +1051,18 @@ public void startFirstTurn(Player goesFirst, Runnable startGameHook) {
                     if (DEBUG_PHASES) {
                         System.out.print("... " + pPlayerPriority + " plays " + chosenSa);
                     }
+
+                    boolean rollback = false;
                     for (SpellAbility sa : chosenSa) {
                         Card saHost = sa.getHostCard();
                         final Zone originZone = saHost.getZone();
 
-                        if (pPlayerPriority.getController().playChosenSpellAbility(sa)) {
+                        if (pPlayerPriority.getController().playChosenSpellAbility(sa) || sa instanceof LandAbility) {
```

Comment:
this seems like a weird addition
might be worth investigating if HumanPlay should really always return false instead